src/components/PageSectionShop.tsx

```diff
@@ -61,7 +61,14 @@ const Shop: FC<FollowUsProps> = ({ className, content }) => {
             <ShirtDesignWrapper>
               <img src={tShirtPlaceholder} width="70%" style={{ margin: 'auto' }} alt="tee-shirt-placeholder" />
               <TeeShirtDesign>
-                <img src={shirtDesign} alt="tee-shirt-design" />
+                {shirtDesigns.map((s) => (
+                  <img
+                    key={s}
+                    src={s}
+                    alt="tee-shirt-design"
+                    style={{ display: shirtDesign === s ? 'block' : 'none' }}
```

Comment:
Did you check that `display: none` is sufficient for not loading the images again? I have the feeling that this will force another HTTP request next time it changes, but I might be wrong! I think that not every browser behaves the same...